source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/org-switcher/row.tsx

```diff
@@ -15,12 +15,14 @@ class Row extends React.Component<IProps> {
   render() {
     const { isActive, organization, onClick } = this.props;
     const attributes = attributesFor(organization);
-    const { name } = attributes;
+    const { name, logoUrl } = attributes;
 
     const className = `
           flex-row align-items-center
           ${isActive ? 'active' : ''}`;
 
+    const logo = logoUrl ? <img src={logoUrl} width='32' /> : '\u00A0';
```

Comment:
:+1:

Reply:
is this the correct size?

Reply:
yes it's

Reply:
screenshot? :)

Reply:
![selecetd](https://user-images.githubusercontent.com/501129/46766625-1f823880-cca8-11e8-9989-ec72eb1571c7.png)
![row](https://user-images.githubusercontent.com/501129/46766626-1f823880-cca8-11e8-9a96-dce6a79cbc23.png)